src/Workspaces/CSharp/Portable/Extensions/SemanticModelExtensions.cs

```diff
@@ -219,25 +220,7 @@ private static ImmutableArray<ParameterName> GenerateNames(IList<string> reserve
             // Otherwise, figure out the type of the expression and generate a name from that
             // instead.
             var info = semanticModel.GetTypeInfo(expression, cancellationToken);
-
-            // If we can't determine the type, then fallback to some placeholders.
```

Comment:
The removed code here was moved to SemanticModelExtensions in the core Workspaces layer for dual use between C# and VB.